app/views/claims/schools/users/remove.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for(:page_title) { sanitize(t(".page_title", user_name: @user.full_name)) } %>
+<% render "claims/support/primary_navigation", current: :organisations %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_school_user_path(@school, @user)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= t(".caption", user_name: @user.full_name) %></span>
+      <h2 class="govuk-heading-l"><%= t(".heading") %></h2>
+
+      <%= govuk_warning_text(text: t(".warning", school_name: @school.name)) %>
+
+      <%= govuk_button_to t(".remove_user"), claims_school_user_path(@school, @user), warning: true, method: :delete %>
+
+      <p class="govuk-body">
+        <%= govuk_link_to(t(".cancel"), claims_school_user_path(@school, @user)) %>
```

Comment:
Just a small thing, but I just fixed all these yesterday. This link should have the `no_visited_state: true` attribute

Reply:
Resolved here: https://github.com/DFE-Digital/itt-mentor-services/pull/301/commits/81bca1e5ea66ffc12ccd1bd0fc39956344bc93bc